provider/terraform.rb

```diff
@@ -115,6 +115,7 @@ def force_new?(property, resource)
     #
     # Note: ?P indicates a Python-compatible named capture group. Named groups
     # aren't common in JS-based regex flavours, but are in Perl-based ones
+    
```

Comment:
We don't need to change their coding style, although it might not be prefect.

Reply:
Resolved.